modules/coreboot

```diff
@@ -42,7 +42,7 @@ CONFIG_COREBOOT_CONFIG ?= config/coreboot-$(BOARD).config
 # Ensure that touching the config file will force a rebuild
 $(build)/$(coreboot_dir)/.configured: $(CONFIG_COREBOOT_CONFIG)
 
-EXTRA_FLAGS := -fdebug-prefix-map=$(pwd)=heads -gno-record-gcc-switches -Wno-error=packed-not-aligned
+EXTRA_FLAGS := -fdebug-prefix-map=$(pwd)=heads -gno-record-gcc-switches -Wno-error=packed-not-aligned -Wno-error=address-of-packed-member
```

Comment:
@MrChromebox This breaks builds for non coreboot 4.15 boards.

Better approach is to define per coreboot version and leave the generic line if not specified
```

diff --git a/modules/coreboot b/modules/coreboot
index 10cc9088..806ee608 100644
--- a/modules/coreboot
+++ b/modules/coreboot
@@ -29,6 +29,7 @@ else ifeq "$(CONFIG_COREBOOT_VERSION)" "4.15"
         coreboot_hash := 20e6aaa6dd0eaec7753441c799711d1b4630e3ca709536386f2242ac2c8a1ec5
         coreboot-blobs_hash := c0e2d8006da226208ba274a44895d102cb2879cf139cc67bba5f62e67b871f6d
         coreboot_depends := $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
+       EXTRA_FLAGS := -fdebug-prefix-map=$(pwd)=heads -gno-record-gcc-switches -Wno-error=packed-not-aligned -Wno-error=address-of-packed-member
 else
        $(error "$(BOARD): does not specify coreboot version under CONFIG_COREBOOT_VERSION")
 endif 
@@ -47,7 +48,7 @@ CONFIG_COREBOOT_CONFIG ?= config/coreboot-$(BOARD).config
 # Ensure that touching the config file will force a rebuild
 $(build)/$(coreboot_dir)/.configured: $(CONFIG_COREBOOT_CONFIG)
 
-EXTRA_FLAGS := -fdebug-prefix-map=$(pwd)=heads -gno-record-gcc-switches -Wno-error=packed-not-aligned -Wno-error=address-of-packed-member
+EXTRA_FLAGS :? -fdebug-prefix-map=$(pwd)=heads -gno-record-gcc-switches -Wno-error=packed-not-aligned
 
 coreboot_configure := \
        mkdir -p "$(build)/$(coreboot_dir)" \

```

Reply:
@tlaurion makes sense to me, I'll adjust / re-push when back at my desk next week